docs/tutorials/kserve-basic.md

```diff
@@ -80,13 +79,13 @@ Use the example workflow definition to create a workflow in FuseML:
 fuseml workflow create workflows/mlflow-e2e.yaml
 ```
 
-**6.** Assign the codeset to the workflow
+## 6. Assign the workflow to the codeset
```

Comment:
Not sure I agree with you here. And I'm not the only one, remember https://github.com/fuseml/fuseml/issues/230 :-)

Reply:
Well, I still think that workflow is assigned to codesets 😄 ... We never reached a conclusion, so the minimum we can do is be consistent, and that change make it consistent with https://github.com/flaviodsr/docs/blob/update_docs/docs/tutorials/seldon-core.md?plain=1#L107